manifests/0000_61_openshift-apiserver-operator_03_version-mapping.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
   namespace: openshift-apiserver-operator
   name: version-mapping
 data:
-  "0.0.0_version_hypershift": "quay.io/openshift/origin-hypershift:v4.0"
-  "0.0.0_version_cluster-openshift-apiserver-operator": "docker.io/openshift/origin-cluster-openshift-apiserver-operator:v4.0"
+  "0.0.1-snapshot-openshift": "quay.io/openshift/origin-hypershift:v4.0"
```

Comment:
You can't do this.  We don't have a version for openshift.

Reply:
Would suggest `openshift-0.0.1-snapshot` or `0.0.1-snapshot_openshift` depending on your use case.